AntennaTracker/Parameters.h

```diff
@@ -114,6 +114,10 @@ class Parameters {
         //
         k_param_command_total = 220,
 
+        // misc
+        k_param_scan_speed_yaw,
```

Comment:
this is good, but set it to k_param_scan_speed_yaw = 225 just in case the waypoints need more. Might as well also move the k_param_gcs_pid_mask into this area too